packages/react-examples/src/react/Nav/Nav.Basic.Example.tsx

```diff
@@ -13,6 +13,7 @@ const navStyles: Partial<INavStyles> = {
 
 const navLinkGroups: INavLinkGroup[] = [
   {
+    name: 'Group',
```

Comment:
I don't think we should include the group name in the default example, since not all navs will have group names.

But it would be good to add an example of a non-collapsible link in this example, and possibly add a non-collapsible group in another example that has multiple groups.